### PR TITLE
Picnic CSS

### PR DESCRIPTION
**Pull request by blinkythebear**
Picnic CSS is an invasive CSS library to get your style started.

### CODE REVIEW

File: files/picnicss/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Francisco Presencia"
+github = "https://github.com/picnicss/picnic"
+homepage = "http://www.picnicss.com/"
+description = "Picnic CSS is an invasive CSS library to get your style started."
+mainfile = "picnic.min.js"
```
Comment (jimaek) [inline files/picnicss/info.ini]: maybe you mean min.css?

File: files/picnicss/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+	"packageManager": "github",
+	"name": "picnicss",
+	"repo": "picnicss/picnic",
+	"files": {
+		"basePath": "releases",
+		"include": ["picnic.css", "picnic.min.css"]
```
Comment (jimaek) [inline files/picnicss/update.json]: Lets use this instead `"picnic.min.css", "plugins.min.css"`

**Comment by blinkythebear** [comment]
All fixed!

**Comment by jimaek** [comment]
Thank you

**Pull request merged**